### DISCUSSION TITLE
part-ii/2015-117/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#382] [Question comments]
Part II 2015 Probability and Measure: Paper 2, Section II, J 

https://questions.tripos.org/part-ii/2015-117/

**Comment by ArthurConmy** [comment]
Apply Holder for first two parts.

For iii), I believe you're supposed to split the integral into the region between $y=x$ and $y=x+h$ and then the regions to the left and right of this. The middle region has integral converging to zero because $ \int_0^\varepsilon x^{-2\alpha}dx \to 0 $ and the tail regions have integral converging to zero because it suffices to show

$$J := \int_0^\infty ((x+h)^{-\alpha} - x^{-\alpha} )^2 dx \to 0$$ 

but by Cauchy-Schwarz with second function $x^{-\alpha}$, 

$$ J \le \left(  \int_0^\infty x^{-2\alpha} dx \right)^{-1} \times \left( \int_0^\infty x^{-\alpha}((x+h)^{-\alpha} - x^{-\alpha} )dx \right)^{2} $$

and because the latter term is dominated by $ 2x^{-2\alpha} $, by dominated convergence this tends to 0.

Would be curious if there's a quicker way to get the last part.
